### PR TITLE
[Packaging] Fix missing quote

### PR DESCRIPTION
**Pull request by troopa81**
## Description

Fix missing quote in script used to build Windows package.

### CODE REVIEW

File: ms-windows/osgeo4w/package.cmd [hunk 1]
```diff
@@ -28,7 +28,7 @@ if "%PACKAGENAME%"=="" goto usage
 if "%ARCH%"=="" goto usage
 if not "%SHA%"=="" set SHA=-%SHA%
 if "%SITE%"=="" set SITE=qgis.org
-if "%BUILDNAME%=="" set BUILDNAME=%PACKAGENAME%-%VERSION%%SHA%-Release-VC14-%ARCH%
```
Comment (m-kuhn) [inline ms-windows/osgeo4w/package.cmd:31]: Was the lineending change on purpose?

**Comment by jef-n** [comment]
cherry-picked cd141490ec instead.

File: ms-windows/osgeo4w/package.cmd (hunk 1 above)
Comment (jef-n) [reply ms-windows/osgeo4w/package.cmd:31]: looks like an artifact from git - package.cmd has CRLF endings.

**Comment by troopa81** [comment]
thanks @jef-n

**Pull request closed without merging**